test/data/build_buildah_cr_custom_context+dockerfile.yaml

```diff
@@ -5,11 +5,11 @@ metadata:
   name: buildah-custom-context-dockerfile
 spec:
   source:
-    url: https://github.com/SaschaSchwarze0/npm-simple
-    contextDir: renamed
+    url: https://github.com/shipwright-io/sample-nodejs
+    contextDir: docker-build
   strategy:
     name: buildah
     kind: ClusterBuildStrategy
-  dockerfile: RenamedDockerfile
+  dockerfile: Dockerfile
```

Comment:
Hereby, we are removing the test for an alternative file name of the Dockerfile.

Reply:
I can add an asset with `RenamedDockerfile` in https://github.com/shipwright-io/sample-nodejs

Reply:
Makes sense.